src/simple-pg/sql/select.js

```diff
@@ -0,0 +1,36 @@
+import sql, { join, raw, empty } from 'sql-template-tag';
+// import debug from 'debug';
+// const log = debug('graffy:pg:select');
+
+const MAX_LIMIT = 4096;
+
+export function selectByArgs(args, options, limit = MAX_LIMIT) {
+  const { table } = options;
+  const where = options;
```

Comment:
I think you mean `whereConditions(args)`.

Taking a step back, I think you should skip `selectByArgs` in this version. Just support read and write through IDs to unblock Harish and Nisheet.

Incidentally, your `whereConditions` function is too limited in usefulness, so I would suggest looking at the `getArgSql` function from the old `pg` and try to port it here, which constructs the WHERE, ORDER BY and LIMIT clauses.